app/components/placement/summary_component.html.erb

```diff
@@ -0,0 +1,75 @@
+<li class="app-search-results__item">
+  <h2 class="app-search-result__item-title">
+    <%= govuk_link_to("", no_visited_state: true) do %>
```

Comment:
```suggestion
    <!-- TODO: link to placement details -->
    <%= govuk_link_to("", no_visited_state: true) do %>
```

Reply:
Didn't think it was necessary as https://github.com/DFE-Digital/itt-mentor-services/pull/541

Reply:
Resolved in [3d9756d](https://github.com/DFE-Digital/itt-mentor-services/pull/531/commits/3d9756d9ceb6973fc9ea3d3d45afe6156d4d6e41)